### PR TITLE
I've added the homework for context_manager

### PR DESCRIPTION
**Pull request by Munteanu-Alexandru**
Updating context manager homework

### CODE REVIEW

File: lesson_19/lesson_19_turcu_alexandru/context_manager.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+let any other exceptions propagate outside of the context manager.
+
+Implement the context manager by using both approaches: by using class and by using @contextmanager
+
+Write examples that prove the functionality for all the handled exceptions and for one exception that is not handled.'''
+import contextlib
+
+
+@contextlib.contextmanager
+def some_exceptions(context):
```
Comment (Liitle) [inline lesson_19/lesson_19_turcu_alexandru/context_manager.py:12]: it is ok to have an argument to your context manager. Remember that what you are yielding, in this case, the `context` is what is attributed to the variable after as
```
whit some_exceptions(context) as context 
```

when you use the context manager you don't have anything after `as`

File: lesson_19/lesson_19_turcu_alexandru/context_manager.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+class SomeException:
+
+    def __enter__(self):
+        print('Entering the exception manager..')
+
+    def __exit__(self, exc_type, exc_value, traceback):
+        if exc_type is IndexError:
+            print(f'Class => Invalid index: {exc_value}', traceback)
+            handled = True
```
Comment (Liitle) [inline lesson_19/lesson_19_turcu_alexandru/context_manager.py:36]: nice use `handled` variable. Good job ! 😄

**Review by Liitle — APPROVED** [review]
Good work. If you got the idea behind what the variable after `as`  is doing we can merge this PR. Let me know in a comment

File: lesson_19/lesson_19_turcu_alexandru/context_manager.py (hunk 1 above)
Comment (Munteanu-Alexandru) [reply lesson_19/lesson_19_turcu_alexandru/context_manager.py:12]: You mean "with SomeException() as context: " right? So its not ideal to have an argument to my context manager. 
Thank you, I will be better in the future.

**Pull request merged**